build/root/WORKSPACE

```diff
@@ -8,9 +8,9 @@ http_archive(
 
 http_archive(
     name = "io_kubernetes_build",
-    sha256 = "007774f06536059f3f782d1a092bddc625d88c17f20bbe731cea844a52485b11",
-    strip_prefix = "repo-infra-97099dccc8807e9159dc28f374a8f0602cab07e1",
-    urls = mirror("https://github.com/kubernetes/repo-infra/archive/97099dccc8807e9159dc28f374a8f0602cab07e1.tar.gz"),
+    sha256 = "7b514fdc635d6f5691422eaa52f686b1be4c146f55e28fa71ff4d7c1300a42f4",
+    strip_prefix = "kubernetes-repo-infra-9e6ce36f59d4a39360080ff4cc13c1b6db172a58",
+    urls = mirror("https://github.com/ixdy/kubernetes-repo-infra/archive/9e6ce36f59d4a39360080ff4cc13c1b6db172a58.tar.gz"),
```

Comment:
TODO(@ixdy): fix before merge

Reply:
what's going on here?

Reply:
this is my fork containing kubernetes/repo-infra#72. once that merges, I'll need to switch this to latest commit from the repo-infra repo.

Reply:
this is fixed now